mm/dynamic/c/Magazine.c

```
@@ -0,0 +1,448 @@
+        sc_depot->partial_magazines = mag;
+    } else { // Empty: return to magazine pool
+        FreeMagazine(mag);
+    }
+}
+
+
+// =================================================================================================
+// Other Public API Functions (Stubs or Simple Implementations)
+// =================================================================================================
+
+void* MagazineAllocate(size_t num, size_t size) {
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'MagazineAllocate' of similar type ('int') are easily swapped by mistake

(bugprone-easily-swappable-parameters)